tools/sz_repo_cli/lib/src/commands/src/build_android_command.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:sz_repo_cli/src/common/common.dart';
+
+final _androidStages = [
+  'stable',
+  'alpha',
+];
+
+/// The different flavors of the Android app.
+final _androidFlavors = [
+  'prod',
+  'dev',
+];
+
+/// The different output types of the Android app.
+final _androidOutputType = [
+  'appbundle',
+  'apk',
+];
+
+class BuildAndroidCommand extends Command {
+  final SharezoneRepo _repo;
+
+  BuildAndroidCommand(this._repo) {
+    argParser
+      ..addOption(
+        releaseStageOptionName,
+        abbr: 's',
+        allowed: _androidStages,
+        defaultsTo: 'stable',
+      )
+      ..addOption(
+        outputTypeName,
+        help: 'The type of output type, either "appbundle" or "apk".',
+        allowed: _androidOutputType,
+        defaultsTo: 'appbundle',
+      )
+      ..addOption(
+        buildNumberOptionName,
+        help: '''An identifier used as an internal version number.
+Each build must have a unique identifier to differentiate it from previous builds.
+It is used to determine whether one build is more recent than
+another, with higher numbers indicating more recent build.
+When none is specified, the value from pubspec.yaml is used.''',
+      )
+      ..addOption(
+        flavorOptionName,
+        allowed: _androidFlavors,
+        help:
+            'The flavor to build for. At the moment only "prod" is supported.',
+        defaultsTo: 'prod',
+      );
```

Comment:
Why is only prod supported right now? (Please document)  
And if only prod is supported right now I would probably throw an Exception in the `run` method if some other flavor is passed.

(I ran `build appbundle --flavor dev` and it worked on my machine, is this just outdated?)

Reply:
How I forgot to remove "At the moment only "prod" is supported."